src/irmin/object_graph.mli

```diff
@@ -48,6 +48,7 @@ module type S = sig
       {b Note:} Both [min] and [max] are subsets of [n]. *)
 
   val iter :
+    ?cache_size:int ->
```

Comment:
I think I'd expect `?cache_size:None` to correspond to the old behaviour: i.e. a cache of unbounded size. Perhaps worth a comment and a CHANGES entry.